lib/nxt_state_machine/state_machine.rb

```diff
@@ -75,6 +75,7 @@ def any_state
       states.values.map(&:enum)
     end
 
+    alias_method :all_states_except, :any_state_except
```

Comment:
It's also just good practice to define `alias_method` directly below the method you're aliasing (e.g. line 84) (just to make it a little easier for searching)